CONTRIBUTING.md

```diff
@@ -1,20 +1,33 @@
 # Nirvana Contributor Guide
 
-Welcome to Nirvana! This document shows you how to contribute to the code base. Plaese file issues or pull requests if you find something is missing or incorrect.
+Welcome to Nirvana! This document shows you how to contribute to the code base. Plaese file issues or pull requests if
+you find anything missing or incorrect.
 
 ### Fork && Clone
 
-First, you can fork this repository via the `Fork` button on the top of this page (If you like Nirvana, please star it :).  
-After that, you can immediately clone `https://github.com/<your-account-name>/nirvana` to your local path `$GOPATH/src/github.com/caicloud/nirvana`.  
-There are some environment requirements:  
+First, you can fork this repository via the `Fork` button on the top of this page (If you like Nirvana, please star it :)).
+Then you can clone Nirvana to your local path:
+
+```
+$ mkdir -p $GOPATH/src/github.com/nirvana
+$ go get github.com/caicloud/nirvana
+
+# Avoid push to caicloud/nirvana
+$ git config push.default nothing
+
+$ git remove add $USER git@github.com:$USER/nirvana.git
```

Comment:
`$USER` is the os user which may be not equal to the github account.

Reply:
this is just copy/paste from kubernetes & cyclone ..